plugins/ingest-attachment/build.gradle

```diff
@@ -1,3 +1,5 @@
+import org.elasticsearch.gradle.BuildPlugin
```

Comment:
This import doesn't seemed to be used. It is also in the wrong place (before the license comment).